manifests/init.pp

```diff
@@ -0,0 +1,69 @@
+# Class: lastpass
+#
+# This class installs the LastPass CLI and provides functions to interact with it.
+
+class lastpass (
+  $package  = $lastpass::params::package,
+  $home     = '/root/.lpass',
+  $agent_timeout = 3600,
+  $username = undef,
+  $password = undef,
+) inherits lastpass::params {
+
+  package { $package:
+    ensure => present,
+  }
+
+  file { '/usr/local/bin/lpasspw':
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+    source => "puppet:///modules/${module_name}/lpasspw",
+  }
+
+  file { $home:
+    ensure => directory,
+    mode   => '0600',
+  }
+  file { "${home}/pw":
+    ensure  => file,
+    mode    => '0400',
+    content => inline_template('<%= @password %>'),
+  }
+
+  profiled::script { 'lpass.sh':
```

Comment:
profiled scripts are usually for logging in real people, but I guess its being picked up b/c the lastpass cli is running interactively. At any case I think we should consider this alternative. At the very least, it reduces dependencies.

One of these should be good:
http://stackoverflow.com/questions/9357132/right-way-to-pass-environment-variables-to-exec-shell-command